test/js/inline-style-property-map.js

```diff
@@ -0,0 +1,16 @@
+suite('Inline StylePropertyMap', function() {
+  setup(function() {
+    this.element = document.createElement('div');
+    document.documentElement.appendChild(this.element);
```

Comment:
It's more common to write

document.body.appendChild(element)

Otherwise the element ends up going after the body, which is a little bit strange.